src/themes/DefaultTheme.js

```diff
@@ -1,17 +1,43 @@
+import { lightenDarkenColor, convertToRgb } from './helpers';
+
 const colors = {
```

Comment:
Consider changing colors names according to http://chir.ag/projects/name-that-color